server/src/immich/api-v1/asset/asset.controller.ts

```diff
@@ -139,12 +137,11 @@ export class AssetController {
   @ApiOkResponse({ content: { 'application/octet-stream': { schema: { type: 'string', format: 'binary' } } } })
   getAssetThumbnail(
     @AuthUser() authUser: AuthUserDto,
-    @Headers() headers: Record<string, string>,
-    @Response({ passthrough: true }) res: Res,
+    @Response() res: Res,
     @Param() { id }: UUIDParamDto,
     @Query(new ValidationPipe({ transform: true })) query: GetAssetThumbnailDto,
   ) {
-    return this.assetService.getAssetThumbnail(authUser, id, query, res, headers);
+    this.assetService.getAssetThumbnail(authUser, id, query, res);
```

Comment:
Maybe this function should be renamed given it doesn't actually return anything now. serveAssetThumbnail?